### ISSUE TITLE
lumino: fix vertical overflow

### ISSUE THREAD

**Issue by oliver-sanders** [cylc/cylc-ui#358]
Follow-on to #325 

I think the workflow component (or possibly some interaction of this and the toolbar / footer) is causing the page to overflow vertically. As a result the scrollbar is appearing and some content is cut off. By fiddling about with tabs it is also possible to make the page grow vertically.

**Pull requests welcome!**

**Comment by hjoliver** [comment]
I figured out the reason for this, but not the solution.

In `src/styles/index.scss`:
```scss
.theme--light.v-application {
  //...
  #core-view {
    min-height: calc(100vh - 82px);
  }
}
```
82px is the height of the footer.  So this is meant to push the footer down to the bottom of the view port.

In the workflows component, the **header** (48px high) is outside of #core-view.  So this would work, but it screws up other components that don't have a header:
```scss
.theme--light.v-application {
  //...
  #core-view {
    min-height: calc(100vh - 82px - 48px);
  }
}

**Comment by hjoliver** [comment]
The dashboard has a similar vertical space issue (causing the browser scroll bar to appear) for a different reason. There is no header, but the #core-view element is pushed down by 16px.  This seems to be because the element immediately inside it has a 16px margin.
```css
.v-application .mt-4 {
    margin-top: 16px !important;
}
```

**Comment by hjoliver** [comment]
Possible solutions:
- if we need a footer, the `100vh - footer-height` sizing has to be used for a box that contains *the entire page content except for the footer*  (which currently isn't the case for the dashboard or the workflows component).
- OR we can just delete the footer and forget about this problem

(@oliver-sanders just suggested deleting the footer, and I tend to agree we don't really need it. Copyright info can be hidden away somewhere else, and "status bar"-like info is going to go in the workflow component header)

**Comment by hjoliver** [comment]
What's your opinion @kinow ?

**Comment by kinow** [comment]
I think deleting the footer is the simplest solution. There's already a possible place to have the footer info (bottom of the left sidebar).

**Comment by hjoliver** [comment]
OK cool, I'll assign myself and do that, as a last thing before going home tonight (working late as I have a short day tomorrow).

**Comment by hjoliver** [comment]
Closed by #361
